includes/class-sensei-lesson.php

```diff
@@ -282,6 +283,48 @@ public function meta_box_save ( $post_id ) {
 		} // End If Statement
 	} // End meta_box_save()
 
+	/**
+	 * When course lessons are being ordered by the user,
+	 * and a new published lesson has not been added to
+	 * course lesson order meta, add it last.
+	 *
+	 * Hooked into `post_save`
+	 *
+	 * @access public
+	 * @param int $lesson_id
+	 * @return void
+	 */
+	public function add_lesson_to_course_order( $lesson_id = 0 ) {
+		$lesson_id = intval( $lesson_id );
+
+		if ( empty( $lesson_id ) ) {
```

Comment:
Is it assumed that if `$lesson_id === 0`, that this is a failure? Just checking since that means this method fails by default and we probably should force a value to be provided.